test/ring_logger_test.exs

```diff
@@ -432,6 +432,18 @@ defmodule RingLoggerTest do
     assert {:error, :enoent} == RingLogger.save("/a/b/c/d/e/f/g")
   end
 
+  test "can retrieve config" do
+    config = %{
+      buffer: {[], []},
+      clients: %{},
+      index: 0,
+      max_size: 10,
```

Comment:
The server config is only `:max_size`. The other fields are internal state that I think we want to keep away from users.